frontend/src/components/common/SideNav.vue

```diff
@@ -53,9 +55,9 @@ const props = defineProps({
                         >
                             <Icon
                                 class="p-sidenav-color-icon p-sidenav-padding-icon"
```

Comment:
Hi Jota, I think we're now using a different way to handle the icon class and style, let's update this line as well. So change it to:
```
:class='custom-sidenav-icon '
```
I checked this two classname, they are in the stylesheet in the common theme repo, let's remove them from there. And we don't need to use two classname, one is good enough to include both padding and color change.
```
.custom-sidenav-icon {
    margin-right: 25px;
    color: map.get(lightTheme.$light-theme, 'interactive');
}
```

Reply:
Just did, thanks!

Reply:
thanks so much Jota!!